x-pack/plugins/event_log/server/event_logger.test.ts

```diff
@@ -102,16 +102,16 @@ describe('EventLogger', () => {
       event: { provider: 'test-provider', action: 'a' },
     });
 
-    const ignoredTimestamp = '1999-01-01T00:00:00Z';
```

Comment:
Because we are now "post-dating" the execute event log docs, we need to allow the timestamp to be overridable  when we merge fixed and initial event fields - previously you could **NOT** override it.  That logic is in the file below, `x-pack/plugins/event_log/server/event_logger.ts`